spec/fixtures/files/offline_vm_noselect.yml

```diff
@@ -0,0 +1,33 @@
+:apiVersion: kubevirt.io/v1alpha1
```

Comment:
Is this really a yaml file? there are always ":" prefixes

Reply:
As above. It is used for specs (tests) only.